masu/database/provider_auth_db_accessor.py

```diff
@@ -23,16 +23,17 @@
 class ProviderAuthDBAccessor(KokuDBAccess):
     """Class to interact with the koku database for Provider Authentication Data."""
 
-    def __init__(self, auth_id, schema='public'):
+    def __init__(self, auth_id=None, provider_resource_name=None):
```

Comment:
It seems unnecessary to drop the `schema` parameter. Unless there's a compelling reason to drop it, I'd suggest just adding on `provider_resource_name` as a new parameter without replacing the `schema` parameter.

Reply:
The compelling reason is that there should never be a case in our current setup where we should allow any schema other than public. Passing in another schema would result in an error.

Reply:
Yeah @adberglund got it.  This database table will always be in the public schema

Reply:
Okay, in that case the parent class shouldn't have it as a parameter at all, then.

If we're going to remove it, let's remove it from the entire hierarchy.

Reply:
Okay, in that case the parent class shouldn't have it as a parameter at all, then.

If we're going to remove it, let's remove it from the entire hierarchy.

Reply:
No.  It's still needed for `KokuDBAccess`.  For example. the report database accessors inherit from `KokuDBAccess` and they supply the appropriate customer tenant schema.